package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "yarn run devcommands && yarn run build:css --watch",
     "devcommands": "yarn run moveimages && yarn run build:css",
     "build": "yarn run devcommands && postcss --use autoprefixer --output dist/style.css dist/style.css",
-    "moveimages": "cp -r assets/images/* dist/",
+    "moveimages": "mkdir dist && cp -r assets/images/* dist/", // this is not setup properly to handle yarn run dev restarts as it will fail since the dist folder already exists.
```

Comment:
This is invalid syntax.

Reply:
Yes sorry I forgot to remove comment as I wanted to leave a note on the PR.